command/genesis/utils.go

```diff
@@ -81,3 +90,83 @@ func fillPremineMap(
 
 	return nil
 }
+
+type GenesisTarget struct {
+	Account *wallet.Account
+	NodeID  string
+}
+
+func ReadValidatorsByRegexp(dir, prefix string) ([]GenesisTarget, error) {
+	if dir == "" {
+		dir = "."
+	}
+
+	files, err := ioutil.ReadDir(dir)
+	if err != nil {
+		return nil, err
+	}
+
+	validCnt := 0
+
+	for _, file := range files {
+		if file.IsDir() && strings.HasPrefix(file.Name(), prefix) {
+			files[validCnt] = file
+			validCnt++
+		}
+	}
+
+	files = files[0:validCnt] // files is local variable, gc will collect everything after function returns
+
+	// we must sort files by number after the prefix not by name string -> eq test-dir-10 should be larger than test-dir-4
```

Comment:
Please refactor this comment to be more descriptive, it shouldn't include examples. If you want to demonstrate examples - write a test for these specific values

Reply:
@igorcrevar Fixed where?